meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk.bb

```diff
@@ -10,7 +10,7 @@ SUPERVISOR_REPOSITORY_x86 = "resin/i386-supervisor"
 SUPERVISOR_REPOSITORY_x86-64 = "resin/amd64-supervisor"
 SUPERVISOR_REPOSITORY_intel-quark = "resin/i386-nlp-supervisor"
 
-SUPERVISOR_TAG ?= "v6.4.2"
+SUPERVISOR_TAG ?= "multicontainer"
```

Comment:
Is this a valid tag? Do we not use versioning anymore?

Reply:
It's not a valid tag for production. I think this PR is so that we can build an image for the multicontainer staging environment, right @agherzan?

Reply:
If so, this needs to be in the description or in a comment somewhere

Reply:
This is a temporary tag. As you can see the PR is a WIP.

Reply:
Okay, got it. I think WIP PRs should not have people assigned to them for review

Reply:
Fair